csharp/downgrades/15b989afd2bfc4743536fdb0958c1d8177a32600/upgrade.properties

```diff
@@ -0,0 +1,2 @@
+description: Add unique constraint on preprocessor directive and compilation pairs
```

Comment:
Do we need a downgrade script that picks a unique preprocessor directive?

Reply:
I don't think we need one, not having a unique value will result in DB inconsistencies, but I think that's acceptable.